bundles/EcommerceFrameworkBundle/PaymentManager/Payment/PayU.php

```diff
@@ -195,7 +195,7 @@ protected function setProducts(array $items): array
             $product = $item->getProduct();
 
             $products[$key]['name'] = $product->getName();
-            $products[$key]['unitPrice'] = (string) round($product->getOSPrice()->getAmount()->asNumeric(), 2) * 100;
+            $products[$key]['unitPrice'] = round($product->getOSPrice()->getAmount()->asNumeric(), 2) * 100;
```

Comment:
It seems that PayU expects this as a string ... same for `totalAmount` , see: 
http://developers.payu.com/en/restapi.html#order_data_retrieve

Reply:
Fixed it already 😉

Reply:
Yes, than it have to be in brackets 👍